src/switchroot/ostree-prepare-root.c

```diff
@@ -83,8 +83,9 @@ const char *config_roots[] = { "/usr/lib", "/etc" };
 #define SYSROOT_KEY "sysroot"
 #define READONLY_KEY "readonly"
 
-// The kernel argument we support to configure composefs.
-#define OT_COMPOSEFS_KARG "ot-composefs"
+#define COMPOSEFS_KEY "composefs"
+#define ENABLED_KEY "enabled"
+#define KEYFILE_KEY "keyfile"
```

Comment:
I originally paused here for a second in confusion because we're parsing a "keyfile" from a (G)"keyfile".  Perhaps `keypath` for the key key?